addons/website/static/src/xml/website.gallery.xml

```diff
@@ -47,12 +53,12 @@
         ========================================================================
     -->
     <t t-name="website.gallery.slideshow.lightbox">
-        <div role="dialog" class="modal o_technical_modal fade" aria-labbelledby="Image Gallery Dialog">
-            <div class="modal-dialog modal-lg" role="Picture Gallery"
-                t-attf-style="min-width: #{dim.min_width}px ; min-height: #{dim.min_height}px ; max-width: #{dim.max_width}px ; max-height: #{dim.max_height}px ; height: #{dim.height}px ;">
-                <div class="modal-content">
-                    <main class="modal-body o_slideshow">
-                        <button type="button" class="close" data-dismiss="modal" style="position: absolute; right: 12px; top: 10px;"><span role="img" aria-label="Close">&amp;times;</span><span class="sr-only">Close</span></button>
+        <div role="dialog" class="modal o_technical_modal fade o_gallery_lightbox p-0" aria-labbelledby="Image Gallery Dialog">
+            <div class="modal-dialog m-0" role="Picture Gallery"
+                t-attf-style="">
+                <div class="modal-content bg-transparent">
+                    <main data-vcss="001" class="modal-body o_slideshow bg-transparent">
```

Comment:
Even the old snippets will use the updated template as it is a static one. So they will all use the 001 here. I think you can remove the code from the 000.scss.

Reply:
Actually, in 000.scss, the same code is used by the lightbox and by the slideshow (same indicators,etc.) so I can't remove it.

Reply:
Mmh indeed. I guess we could remove the `.modal:not([data-vcss])` selector but probably not worth it :+1: